### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by vogab** [hyemin-han/BayesFMRI#2]
Hello! I'm very interested in using this package. However, I have troubles getting it to run. I installed the python dependencies in a conda environment and tried to run bayes_select_ui.py from this environment in the anaconda prompt. The gui appears and i'm asked to choose the task to do next. I select bayesian correction. Then, I get this error message:

Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases.

Then I tried to remove python from the "app execution aliases". The gui opens as before, but I get a new error message:

Traceback (most recent call last):
  File "C:\Users\allst\BayesFactorFMRI-master\V1.0.0\bayes_select_ui.py", line 89, in <module>
    bayes_select_ui()
  File "C:\Users\allst\BayesFactorFMRI-master\V1.0.0\bayes_select_ui.py", line 37, in bayes_select_ui
    at_the_end(nexts)
  File "C:\Users\allst\BayesFactorFMRI-master\V1.0.0\bayes_select_ui.py", line 50, in at_the_end
    subprocess.call(["python3", "bayes_correction_ui.py"], cwd="Correction_UI")
  File "C:\Users\allst\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\allst\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\allst\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1440, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

Any thought on how to solve this?

**Comment by hyemin-han** [comment]
Hi, thanks for using BayesFactorFMRI. Unfortunately, the matter seems to be related to your Python and/or Conda installed on Windows... Please refer to this post on StackExchange for some ideas about how to address the issue:

https://stackoverflow.com/questions/65348890/python-was-not-found-run-without-arguments-to-install-from-the-microsoft-store

**Comment by vogab** [comment]
Hi, thanks for the suggestion! In the end, I reinstalled my python and R and the required packages so that I can run them from my windows command line. My python and R seem to work fine, but I have the same issue as before. The gui appears and I'm asked to choose the task to do next. I select bayesian correction. Then, the gui disappears and I get this error message:

C:\Users\allst\BayesFactorFMRI-master\V1.0.0>python bayes_select_ui.py
Traceback (most recent call last):
  File "C:\Users\allst\BayesFactorFMRI-master\V1.0.0\bayes_select_ui.py", line 89, in <module>
    bayes_select_ui()
  File "C:\Users\allst\BayesFactorFMRI-master\V1.0.0\bayes_select_ui.py", line 37, in bayes_select_ui
    at_the_end(nexts)
  File "C:\Users\allst\BayesFactorFMRI-master\V1.0.0\bayes_select_ui.py", line 50, in at_the_end
    subprocess.call(["python3", "bayes_correction_ui.py"], cwd="Correction_UI")
  File "C:\Users\allst\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 389, in call
    with Popen(*popenargs, **kwargs) as p:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\allst\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1024, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\allst\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1509, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified


It looks like there is some problem with the "subprocess", and especially the subprocess.call() function.

Maybe something relevant there:
https://python-forum.io/thread-37845.html

I use python 3.11.3, and R 4.2.3 on a windows 10 machine.

Any idea on how to solve this?

**Comment by vogab** [comment]
PS: following suggestions in the link sent above, I added shell=True in the subprocess.call() line 50 in bayes_select_ui.py.

Then I got this error:
'python3' is not recognized as an internal or external command,
operable program or batch file.

So I changed python3 to python line 50, finally being:
		subprocess.call(["python", "bayes_correction_ui.py"], cwd="Correction_UI",shell=True)

It looks, like it's working now. At least, I reached the next step in which the gui asks to select the working directory. I will see if the following steps work too.
